diff --git a/runtime/library.d.ts b/runtime/library.d.ts
index c5aef502e9786b3519757f7710e614509d8ef1cb..0aceb0f115bf36b0751b8dcb01f23a0db07d9112 100644
--- a/runtime/library.d.ts
+++ b/runtime/library.d.ts
@@ -2689,7 +2689,7 @@ export declare type Return<T> = T extends (...args: any[]) => infer R ? R : T;
 
 declare type Runtime = "edge-routine" | "workerd" | "deno" | "lagon" | "react-native" | "netlify" | "electron" | "node" | "bun" | "edge-light" | "fastly" | "unknown";
 
-declare type RuntimeDataModel = {
+export declare type RuntimeDataModel = {
     readonly models: Record<string, RuntimeModel>;
     readonly enums: Record<string, RuntimeEnum>;
     readonly types: Record<string, RuntimeModel>;
