diff --git a/esm/plugin.js b/esm/plugin.js
index 63e99716a70bad675c0efd34138938871f39643e..63352ff441298e883f1e7fc444a6f8b8630f0d0b 100644
--- a/esm/plugin.js
+++ b/esm/plugin.js
@@ -472,9 +472,9 @@ ${c.files
                 : // Format Others
                     objTypes
                         .map((c) => {
-                        return (`/**\n * ${c.type}\n */
-${options?.format?.includes('object') ? `export ` : ``}` +
-                            `const ${c.type} = {
+                        return (`/**\n * ${c.type}\n */\n` +
+
+                            `export const ${c.type} = {
   ${c.files
                                 .map((key) => {
                                 if (key.strParams) {
